src/advanced-slots.tsx

```diff
@@ -0,0 +1,350 @@
+import * as React from 'react';
+import {sprinkles} from './sprinkles.css';
+import Stack from './stack';
+import * as styles from './advanced-slots.css';
+import {Text2, Text3, Text5, Text8} from './text';
+import {vars} from './skins/skin-contract.css';
+import Inline from './inline';
+import Box from './box';
+import ProgressBar from './progress-bar';
+import classNames from 'classnames';
+
+import type StackingGroup from './stacking-group';
+import type Image from './image';
+import type Tag from './tag';
+import type {RendersNullableElement} from './utils/renders-element';
+
+interface SlotContentProps {
+    title?: string;
+    description?: string | Array<string>;
+}
+
+const SlotContent: React.FC<SlotContentProps> = ({title, description}) => {
```

Comment:
these components can be used out of the slots. Maybe we need to rename this to block?